src/citation/related_models/citation_entry_model.py

```diff
@@ -26,6 +27,13 @@ class CitationEntry(DefaultAuthenticatedModel):
     organization = models.ForeignKey(
         Organization, related_name="created_citations", on_delete=models.CASCADE
     )
+    paper = models.ForeignKey(
```

Comment:
Wonder if we should just link it `rh_uni_doc` instead. 
In case we decide to allow users to cite researchhub contents in the future, it's better to get ahead of this and allow unified document instead of trying to add individual document types.

Reply:
\+ 1